projects/kit/components/input-count/input-count.component.ts

```diff
@@ -233,7 +247,7 @@ export class TuiInputCountComponent
     }
 
     protected getFallbackValue(): number {
-        return 0;
+        return NaN;
```

Comment:
[InputNumber](https://taiga-ui.dev/components/input-number/API) is `null` when empty.
Probably, `InputCount` should have the same behavior 🤔 

<img width="1057" alt="Screenshot 2022-12-27 at 11 41 03" src="https://user-images.githubusercontent.com/35179038/209638597-788b22cd-ab67-4d3d-a1e9-e5d0f0ff599d.png">

Reply:
fixed